src/Sentry/Protocol/UserFeedback.cs

```diff
@@ -43,5 +39,43 @@ public UserFeedback(SentryId eventId, string email, string comments, string? nam
             Email = email;
             Comments = comments;
         }
+
+        public void WriteTo(Utf8JsonWriter writer)
+        {
+            writer.WriteStartObject();
+
+            // Event ID
+            writer.WriteSerializable("event_id", EventId);
+
+            // Name
+            if (!string.IsNullOrWhiteSpace(Name))
+            {
+                writer.WriteString("name", Name);
+            }
+
+            // Email
+            if (!string.IsNullOrWhiteSpace(Email))
+            {
+                writer.WriteString("email", Email);
+            }
+
+            // Comments
+            if (!string.IsNullOrWhiteSpace(Comments))
+            {
+                writer.WriteString("comments", Comments);
+            }
+
+            writer.WriteEndObject();
+        }
+
+        public static UserFeedback FromJson(JsonElement json)
+        {
+            var eventId = json.GetPropertyOrNull("event_id")?.Pipe(SentryId.FromJson) ?? SentryId.Empty;
+            var name = json.GetPropertyOrNull("name")?.GetString() ?? "";
+            var email = json.GetPropertyOrNull("email")?.GetString() ?? "";
```

Comment:
This is nullable, so rather let it be null than give it an empty string.

Also I'm not sure it makes sense to give it an instance full of empty strings if the vallues weren't there (likely a bug since u can't create such instance).

Maybe best to return null.

Reply:
It's not nullable on the class

Reply:
I mean `Name`.

These empty strings are not great. can't we return null and log an error here?
We got a broken file in this case. Sending `""` to Sentry because the value is required is such a hack